src/plugins/intel_cpu/src/docs/README.md

```diff
@@ -14,6 +14,7 @@ Use the following cmake option to enable debug capabilities:
 * [Graph serialization](graph_serialization.md)
 * [Graph transformation disabling](feature_disabling.md#graph-transformations)
 * [Logging](logging.md)
+* [Fusings disabling](fusing_disabling.md)
```

Comment:
Maybe it worths to generalize the name: `Fusings disabling` -> `Graph optimizations disabling`.